Content.Shared/GameTicking/SharedGameTicker.cs

```diff
@@ -144,18 +145,36 @@ public TickerJobsAvailableEvent(Dictionary<NetEntity, string> stationNames, Dict
         }
     }
 
-    [Serializable, NetSerializable]
-    public sealed class RoundEndMessageEvent : EntityEventArgs
+    [Serializable, NetSerializable, DataDefinition]
+    public sealed partial class RoundEndMessageEvent : EntityEventArgs
     {
-        [Serializable, NetSerializable]
-        public struct RoundEndPlayerInfo
+        [Serializable, NetSerializable, DataDefinition]
+        public partial struct RoundEndPlayerInfo
         {
+            [DataField]
             public string PlayerOOCName;
+
+            [DataField]
             public string? PlayerICName;
+
+            [DataField]
+            public NetUserId? PlayerGuid;
```

Comment:
Mark these as `[NotSerialized]` because they don't need to be sent over the network.